BoxContentSDK/BoxContentSDK/BOXContentSDKConstants.m

```diff
@@ -274,6 +282,7 @@
 NSString *const BOXAPIObjectKeyCanSeeManagedUsers = @"can_see_managed_users";
 NSString *const BOXAPIObjectKeyIsSyncEnabled = @"is_sync_enabled";
 NSString *const BOXAPIObjectKeyStatus = @"status";
+NSString *const BOXAPIObjectKeyState = @"state";
```

Comment:
There was an extern that was never defined, probably left over from a refactor/cleanup. It has since been removed. Thanks Jim.